src/data-providers/__tests__/dexie.ts

```diff
@@ -25,6 +24,9 @@ jest.mock('lodash', () => {
   )
 })
 
+// mock getUserRef (firebase's database.ref)
+jest.mock('../../util/getUserRef')
+
```

Comment:
It was mocked because the function `getUserRef` was always expected to return a valid value (an invalid assumption). Now that it returns `Ref | null`, we don't need to mock it